src/index.js

```diff
@@ -1,3 +1,46 @@
+import main from './js/home';
+import menu from './js/menu';
 import contact from './js/contact';
 
-contact();
\ No newline at end of file
+function free() {
+  const item = document.querySelector('body');
+  while (item.firstChild) {
+    item.removeChild(item.firstChild);
+  }
+}
+
+function navbar() {
+  const nav = document.createElement('nav');
+  nav.classList.add('nav');
+  const ul = document.createElement('ul');
+  ul.classList.add('ul');
+
+  const links = ['Home', 'Menu', 'Contact'];
+  for (let i = 0; i < 3; i += 1) {
+    const btn = document.createElement('buttton');
+    btn.innerHTML = links[i];
+    btn.classList.add('li', links[i]);
```

Comment:
- [ ] Please change these and all other regular functions to es6 arrow function syntax.